### PR TITLE
Fix page limit issue,ui improvements, added link page

### PR DESCRIPTION
**Pull request by kingshuk5**
Fixed page limit Issue
added on scroll animation
added 3 bar slider
added link-tree page
added recent weeklies in the header

### CODE REVIEW

**Comment by a2ke5e1** [comment]
image
Fix such that in mobile devices that page is not scrollable horizontally.
Add some padding on hover 
image

**Comment by a2ke5e1** [comment]
There are few issue that I will it later today than merge it.

**Review by a2ke5e1 — APPROVED** [review]

**Comment by NgoEureka** [comment]
ok

**Comment by kingshuk5** [comment]
ok

File: tailwind.config.ts [hunk 1]
```diff
@@ -20,7 +20,7 @@ const config: Config = {
         "body": "#f1f2f3",
       },
       screens: {
-        "2.5xl": "1367px",
```
[the remark this answers is not in the document]
Comment (kingshuk5) [reply tailwind.config.ts]: ok i will take care of it.

**Pull request merged**